util/common.js

```diff
@@ -0,0 +1,27 @@
+const { flatten } = require("flat");
+const { isEqual } = require("lodash");
+
+class CommonUtils {
+  static objectDiff(obj1, obj2) {
+    const flatObj1 = flatten(obj1 || {});
+    const flatObj2 = flatten(obj2 || {});
+    const allKeys = Array.from(
+      new Set(Object.keys(flatObj1).concat(Object.keys(flatObj2)))
+    );
+    return allKeys
+      .filter(key => {
+        if (!isEqual(flatObj1[key], flatObj2[key])) {
```

Comment:
**Minor**: we can change this to return `!isEqual(flatObj1[key], flatObj2[key])` itself right ?

Reply:
Yes I did while debugging, forget to revert.